docs/_data/nav_docs.yml

```diff
@@ -32,6 +32,8 @@
       title: JSX In Depth
     - id: typechecking-with-proptypes
       title: Typechecking With PropTypes
+    - id: loading-data-asynchronously
+      title: Loading Data Asynchronously
     - id: refs-and-the-dom
```

Comment:
Going back to it, I saw that other guides in the same section don't have previous links. I removed the one in this article to keep consistency.